modules/kubernetes/prometheus/charts/prometheus-extras/templates/alertmanager.yaml

```diff
@@ -0,0 +1,46 @@
+{{- if .Values.alertmanager.enabled }}
+---
+apiVersion: monitoring.coreos.com/v1
+kind: Alertmanager
+metadata:
+  name: alertmanager
+spec:
+  replicas: 2
+  alertmanagerConfigSelector:
+    matchLabels:
+      alertmanagerConfig: opsgenie
+---
+apiVersion: v1
+kind: Service
+metadata:
+  name: alertmanager
+spec:
+  ports:
+  - name: web
+    port: 9093
+    protocol: TCP
+    targetPort: web
+  selector:
+    alertmanager: alertmanager
+---
+apiVersion: monitoring.coreos.com/v1alpha1
+kind: AlertmanagerConfig
+metadata:
+  name: config-opsgenie
+  labels:
+    alertmanagerConfig: opsgenie
+spec:
+  route:
+    groupBy: ['job']
+    groupWait: 30s
+    groupInterval: 5m
+    repeatInterval: 12h
+    receiver: 'opsgenie'
+  receivers:
+  - name: 'opsgenie'
+    opsgenieConfigs:
+    - apiURL: 'https://api.eu.opsgenie.com/'
+      apiKey:
+        name: 'opsgenie-config'
+        key: 'apiSecret'
```

Comment:
We should make opsgenie configurable using variables. `apiURL` should be a variable in Helm that we inject from terraform. apiKey.key and apiKey.name should alsoe be variables.